openff/toolkit/topology/molecule.py

```diff
@@ -4586,7 +4586,8 @@ def from_iupac(
             molecule = toolkit.from_iupac(
                 iupac_name,
                 allow_undefined_stereo=allow_undefined_stereo,
-                _cls=cls ** kwargs,
+                _cls=cls,
+                **kwargs,
```

Comment:
This ... probably wasn't working before.

Reply:
This is the only manual change I made (adding a comma).

Reply:
(not blocking) Oy vey. I hate this "registry or wrapper" pattern, so many untested cases! Let's never use this design pattern ever again.